src/reducers/entities/posts.js

```diff
@@ -217,6 +217,66 @@ function currentFocusedPostId(state = '', action) {
     }
 }
 
+function reactions(state = {}, action) {
+    switch (action.type) {
+    case PostTypes.RECEIVED_REACTIONS: {
+        const reactionsList = action.data;
+        const nextReactions = {...(state[action.postId] || {})};
```

Comment:
i'm 0/5 but we could declare this const before the switch so we don't need to declare them every time inside each case